types/bun/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "private": true,
+    "name": "@types/bun",
+    "version": "1.0.9999",
+    "minimumTypeScriptVersion": "4.7",
+    "projects": [
+        "https://bun.sh"
+    ],
+    "devDependencies": {
+        "@types/bun": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Colin McDonnell",
+            "githubUsername": "colinhacks"
+        },
+        {
+            "name": "Jarred Sumner",
+            "githubUsername": "Jarred-Sumner"
+        },
+        {
+            "name": "Ashcon Partovi",
+            "githubUsername": "electroid"
+        },
+        {
+            "name": "Dave Caruso",
+            "githubUsername": "paperdave"
+        }
+    ],
+    "dependencies": {
+        "tsd": "^0.29.0"
```

Comment:
Does the package actually depend on tsd?

Reply:
oops not anymore